### ISSUE TITLE
CWOD-Vampire sheet - INITIATIVE?

### ISSUE THREAD

**Issue by jrichview** [Roll20/roll20-character-sheets#6416]
Does the sheet even support Initiative roll?

**Comment by RinVindor** [comment]
You'll have to be more specific with which Vampire CWOD sheet you mean. Do you mean the V20, Dark Ages V20 or Vampire Revised sheet?

**Comment by jrichview** [comment]
Yep, sorry. I thought this forum was for the older rev only. I'm playing revised.

**Comment by jrichview** [comment]
My only R20 experience thus far is as a player in AD&D 5e. Trying to bring together a VtM Chronicle for revised rules.

**Comment by nmbradley** [comment]
@jrichview I know that this is from a long time ago, but would it be possible to get some greater clarity on this issue:

Do you want someone to add initiative rolling to a sheet?
Is [this](https://github.com/Roll20/roll20-character-sheets/blob/master/CWOD-Vampire/Vampire.png) the sheet in question?

**Comment by nmbradley** [comment]
Thanks for the response. 

If you'd be willing to share the macro when you have time, it might help someone pick up this issue and resolve it!

**Comment by jrichview** [comment]
Sure. Here it is:

`
&{template:default} {{name=Initiative - @{selected|token_name} }}{{@{selected|token_name} Rolls=[[1d10 + @{selected|Dexterity} + @{selected|Wits} &{tracker}]]}}
`

**Comment by jrichview** [comment]
> 
> 
> I think the only reason &{tracker} needs a token to be selected first, is so that when you mouse over the tracker position (with the token's picture next to it), the token itself gets highlighted. Without, it should still be able to roll initiative, it just won't be placed on the turn tracker because there's no token to go with.

That makes sense, of course. But my point is that it should be doable as a button from the character sheet because you can associate the token and the character sheet.

**Comment by Anduh** [comment]
> Sure. Here it is:
> 
> `&{template:default} {{name=Initiative - @{selected|token_name} }}{{@{selected|token_name} Rolls=[[1d10 + @{selected|Dexterity} + @{selected|Wits} &{tracker}]]}}`

Yeah, removing the explicit `selected` part would make the sheet roll better, by not preventing init rolls just bc a token wasn't selected.

Something like this would be a better idea:

`&{template:default} {{name=Initiative - @{character_name} }}{{@{selected|character_name} Rolls=[[1d10 + @{Dexterity} + @{Wits} &{tracker}]]}}`

Referencing the built-in character name of the sheet, which in most cases is the same as the token's name, works much better.
